### PR TITLE
[Flat Theme]: dojo/form styles added 

### PR DESCRIPTION
**Pull request by yiweima**

### CODE REVIEW

**Comment by dylans** [comment]
Thanks @yiweima , we look forward to reviewing this soon!

File: FLAT/dojo/icons/commonIcons_rtl.styl [hunk 1]
```diff
@@ -0,0 +1,19 @@
+//
+// Copyright 2014 Esri
```
Comment (dylans) [inline FLAT/dojo/icons/commonIcons_rtl.styl]: cruft

File: FLAT/dojo/icons/editorIcons_rtl.styl [hunk 2]
```diff
@@ -0,0 +1,35 @@
+//
+// Copyright 2014 Esri
```
Comment (dylans) [inline FLAT/dojo/icons/editorIcons_rtl.styl:2]: cruft

File: FLAT/dojo/Dialog.styl [hunk 3]
```diff
@@ -0,0 +1,308 @@
+/* Dialog
```
Comment (wkeese) [inline FLAT/dojo/Dialog.styl:1]: There are some naming conflicts by just putting everything in a directory called "dojo", like between dijit/Dialog and dojox/widget/Dialog.  (Even if you don't plan to now, someday someone may add a theme for dojox/widget/Dialog in addition to dijit/Dialog.)

Comment (dylans) [inline FLAT/dojo/Dialog.styl:1]: Agreed, I'm thinking he probably needs to have a structure like the following:

/themes/flat/dijit/
/themes/flat/dijit/form/
/themes/flat/dijit/layout/
/themes/flat/dojox/
/themes/flat/dojox/form/
/themes/flat/dojox/layout/
/themes/flat/dojox/widget/

Unless there's a better suggestion.

**Comment by dylans** [comment]
Closed via e4eab64, thanks for the efforts in getting this started!

**Pull request closed without merging**